packages/controller-utils/src/util.ts

```diff
@@ -435,7 +435,7 @@ type EthQueryLike = {
         | [error: null, result: unknown]
     ) => void,
   ) => void;
-};
+} & Record<string, (...args: unknown[]) => void>;
```

Comment:
I get that at runtime, the `ethQuery` argument could be anything, but from a type perspective, don't we want consumers to pass something that has a `sendAsync` method? It seems like the runtime check we're making in `query` is really for JavaScript code and not TypeScript code.

Reply:
Hmm, i think that type would have still required `sendAsync`. Though I have just adjusted the types to reference our top-level eth-query types instead.